src/client/components/app/style.css

```diff
@@ -1,5 +1,9 @@
 @import url('https://fonts.googleapis.com/css?family=Open+Sans:300,400,500');
 
+* {
+	box-sizing: border-box;
```

Comment:
Do you think this will ever come back to bite us? Would libraries ever assume `content-box` is the default and break? Maybe its fine and we'll revisit if its a problem.